tests/e2e/ansible/install_test_deps.yaml

```diff
@@ -33,6 +33,48 @@
                 path: bats-core
                 state: absent
           when: bats_exist.rc != 0
+    - block:
+      - name: Check skopeo is installed
+        shell: command -v skopeo >/dev/null 2>&1
+        register: skopeo_exist
+        ignore_errors: yes
+      - name: Install skopeo
+        shell: |
+          sudo apt-get install -y build-essential libgpgme-dev libassuan-dev libbtrfs-dev pkg-config go-md2man
+          git clone https://github.com/containers/skopeo
+          cd skopeo
+          DISABLE_DOCS=1 make
+          DISABLE_DOCS=1 sudo make install
+    - block:
+      - name: Install kbs test dependencies
+        shell: |
+          sudo apt-get install -y build-essential pkg-config libssl-dev
+    - block:
+      - name: Check rust is installed
+        shell: command -v rustc >/dev/null 2>&1
+        register: rustc_exist
+        ignore_errors: yes
+      - name: Install rust
+        shell: |
+          curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain {{ rust_version }}
+    - block:
+      - name: Create temp trustee directory
+        shell: |
+              mkdir /tmp/trustee
+      - name: Clone trustee repository
+        git:
+          repo: https://github.com/confidential-containers/trustee.git
+          dest: /tmp/trustee
+          clone: yes
+          force: yes
+        retries: 3
+        delay: 10
+      - name: Build and install the kbs client
```

Comment:
It doesn't look like you actually use the `kbs-client` in this test. You can probably skip installing and uninstalling it.